mantle/kola/tests/ostree/unlock.go

```diff
@@ -44,8 +44,8 @@ func init() {
 }
 
 var (
-	rpmUrl  string = "https://raw.githubusercontent.com/projectatomic/atomic-host-tests/master/rpm/aht-dummy-1.0-1.noarch.rpm"
-	rpmName string = "aht-dummy"
+	rpmUrl  string = "https://dl.fedoraproject.org/pub/epel/"
+	rpmName string = "epel-release"
 )
```

Comment:
you'll probably need to remove the `command -v` test below because this RPM doesn't provide any binaries.

Reply:
removed the unnecessary binary related variables